docker-compose.local.yml

```diff
@@ -125,7 +127,7 @@ nginx:
   environment:
     - NGINX_CONFIG_FILE_NAME=nginx_site_http.conf
   ports:
-    # NOTE: The host ports (left of the ":") must match the values in 
+    # NOTE: The host ports (left of the ":") must match the values in
```

Comment:
Sorry about that, my editor removed trailing spaces for Python by default, I just disabled it.